CONTRIBUTING.md

```diff
@@ -37,6 +38,14 @@ expect throughout each step of the process.
       * [Approval and Request Changes Workflow](#approval-and-request-changes-workflow)
     * [Step 10: Landing](#step-10-landing)
   * [Reviewing Pull Requests](#reviewing-pull-requests)
+    * [Review a bit at a time](#review-a-bit-at-a-time)
+    * [Be aware of the person behind the code](#be-aware-of-the-person-behind-the-code)
+    * [Respect the minimum wait time for comments](#respect-the-minimum-wait-time-for-comments)
+    * [Abandoned or Stalled Pull Requests](#abandoned-or-stalled-pull-requests)
+    * [Approving a change](#approving-a-change)
+    * [Accept that there are different opinions about what belongs in Node.js](#accept-that-there-are-different-opinions-about-what-belongs-in-node-js)
```

Comment:
#accept-that-there-are-different-opinions-about-what-belongs-in-node-js ->
#accept-that-there-are-different-opinions-about-what-belongs-in-nodejs